README.md

```diff
@@ -61,7 +70,10 @@ In order to be useful, Anon needs to be told what you want to do to each column
     {
       // Takes a UK format postcode (eg. W1W 8BE) and just keeps the outcode
       // (eg. W1W).
-      "name": "outcode"
+      "name": "outcode",
+      // what field in the json this action needs to be applied. If a field in
+      // the json doesn't have an action defined, then it will be left untouched.
+      "JsonField": "postcode"
```

Comment:
Can we camel-case this to match the rest? eg. `jsonField` instead of `JsonField`

Reply:
it's actually how it should be, that was a typo (probably because of copy&pasting from go code)